WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/TaskManager.cs

```diff
@@ -52,6 +53,50 @@ public partial class DevicePortal
             return processId;
         }
 
+        /// <summary>
+        /// Gets the current running apps on the console based on app processes
+        /// </summary>
+        /// <returns>List of Package Full Names of running apps</returns>
+        public async Task<List<string>> GetRunningAppsAsync()
+        {
+            RunningProcesses appProcesses = await this.GetRunningProcessesAsync();
+            List<string> runningApps = new List<string>();
+
+            foreach (DeviceProcessInfo process in appProcesses.Processes)
+            {
+                // There can be multiple processes per app, so only add an app package name if it doesn't already
+                // exist in the list and it is actually running - processes listed can be in the suspended state
+                if (process.PackageFullName != null && process.IsRunning && !runningApps.Contains(process.PackageFullName))
```

Comment:
Does the contain function call do an exact string match or will it allow for partial string matches?

Reply:
The Java List.contains function does an exact string match on an element - [Documentation](https://docs.oracle.com/javase/7/docs/api/java/util/List.html#contains%28java.lang.Object%29)